doc/modules/mod_jingle_sip.md

```diff
@@ -1,22 +1,22 @@
 ### Module Description
 
-This module enabled Jingle to SIP and SIP to Jingle translator.
-When this module is enalbed, MongooseIM will intercept any iq set jingle stanza with action:
+This module enables Jingle to SIP and SIP to Jingle translation.
+When this module is enabled, MongooseIM will intercept any Jingle IQ set stanza with action:
 * session-initiate
 * session-terminate
 * session-accept
 * transport-info
 
-and translate it to SIP messages with approprite SDP content based on the details in the Jingle stanza.
+and translate it to SIP messages with appropriate SDP content based on the details in the Jingle stanza.
 
 The translation back from SIP to Jingle is done for following SIP messages:
 
-* `INVITE` - with additionall callback for following response codes:
+* `INVITE` - with additional callback for the following response codes:
```

Comment:
1. I know it may be derived from tests sequence diagrams but I'd expect here a description of mappings between specific SIP and Jingle messages.
2. What is "additional callback"?

Reply:
1. OK, I'll try to come up with a clear and readable table for that.
2. There can be many different responses to the INVITE request. These responses are handled in a callback function set for the INVITE request.

Reply:
1. OK
2. Is it configurable? If not, is it only an implementation detail or something a user/client must be aware of in order to use `mod_jingle_sip` properly? I'm sorry if it's a stupid question, may be a consequence of my poor SIP knowledge, but simply by reading this paragraph I don't know how this information affects client-side code or server-side setup.